### PR TITLE
Duplicate Point Check

### PR DESCRIPTION
**Pull request by savannahostrowski**
This check flags Points when there are two or more in the exact same location.

This PR includes logic, documentation, configuration, and unit tests.

### CODE REVIEW

**Comment by savannahostrowski** [comment]
@kkonishi2 If you have anything else to document for other enhancements, please write them here so I can keep track! 😄

**Comment by savannahostrowski** [comment]
@danielduhh Ready for review.

File: docs/checks/duplicatePointCheck.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Duplicate Point Check
+
+#### Description
+This check flags Points when there are two or more in the exact same location.
```
Comment (danielduhh) [inline docs/checks/duplicatePointCheck.md]: It's worth mentioning why this is an issue.

File: src/test/java/org/openstreetmap/atlas/checks/validation/points/DuplicatePointCheckTest.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+import org.openstreetmap.atlas.geography.Segment;
+import org.openstreetmap.atlas.geography.atlas.Atlas;
+import org.openstreetmap.atlas.geography.atlas.packed.PackedAtlasBuilder;
+import org.openstreetmap.atlas.utilities.collections.Iterables;
+import org.openstreetmap.atlas.utilities.random.RandomTagsSupplier;
+
+/**
+ * Test to check whether it picks up the duplicate Points correctly
+ *
+ * @author mcuthbert
```
[the remark this answers is not in the document]
Comment (savannahostrowski) [reply src/test/java/org/openstreetmap/atlas/checks/validation/points/DuplicatePointCheckTest.java]: But it's basically a direct copy of the unit tests that Michael wrote?

File: docs/checks/duplicatePointCheck.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+#### Description
+The Duplicate Point check flags Points in OSM that share the same location. Whether it's a street
+light, viewpoint, or tree, a Point represents something that exists at that location, therefore, we
+do not want points on top of each other. Also, as per the [One feature, one OSM element principle](https://wiki.openstreetmap.org/wiki/One_feature,_one_OSM_element), 
+we don't want instances of duplicate Points that represent the same feature. 
+
+#### Live Example
+The following examples illustrate two cases where there are two or more Points in the exact same location.
+1) This Point [id:538464430](https://www.openstreetmap.org/node/538464430) has two Points in the same
```
Comment (danielduhh) [inline docs/checks/duplicatePointCheck.md]: Not sure what is more problematic, the duplicate point, or that there's a Chili's in Egypt!!!!!

File: config/configuration.json [hunk 4]
```diff
@@ -61,6 +61,14 @@
       "difficulty": "EASY"
     }
   },
+   "DuplicatePointCheck": {
+     "challenge": {
+       "description": "Tasks contain node locations where duplicate points of interest are found.",
+       "blurb": "Duplicate Points",
+       "instruction": "Open your favorite editor and remove one or more of the duplicate points of interest until only one remains."
```
Comment (danielduhh) [inline config/configuration.json]: `instruction` is missing an end comma

**Comment by danielduhh** [comment]
Let it fly

**Pull request closed without merging**